### PR TITLE
Add `licenseKey` to CDN index page.

### PR DESCRIPTION
**Pull request by Mati365**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Internal: Add `licenseKey` to CDN index page. Closes https://github.com/cksource/ckeditor5-commercial/issues/6792

### CODE REVIEW

**Review by Witoso — APPROVED** [review]

File: scripts/release/assets/zip/index.html [hunk 1]
```diff
@@ -27,39 +27,40 @@
...
-        </script>
-        <!-- A friendly reminder to run on a server, remove this during the integration. -->
-        <script>
-		        window.onload = function() {
-		            if ( window.location.protocol === "file:" ) {
-		                alert( "This sample requires an HTTP server. Please serve this file with a web server." );
-		            }
-		        };
+					],
+					licenseKey: 'GPL'
```
Comment (Witoso) [inline scripts/release/assets/zip/index.html:47]:
```suggestion
					licenseKey: 'GPL' // Or <YOUR_LICENSE_KEY>.
```

Comment (Mati365) [reply scripts/release/assets/zip/index.html:47]: k, I'll update another PR with that.

Comment (godai78) [reply scripts/release/assets/zip/index.html:47]: SHould we make it in this order?

Comment (Mati365) [reply scripts/release/assets/zip/index.html:47]: imho, it's much more user-friendly (if you ask about `GPL` and comment order)

Comment (godai78) [reply scripts/release/assets/zip/index.html:47]: Sure, but we have it the other way round everywhere else.

Comment (Mati365) [reply scripts/release/assets/zip/index.html:47]: Agh, I'll fix that in the second PR.

**Pull request merged**